rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/helm/HelmBakeManifestService.java

```diff
@@ -12,7 +12,8 @@
 @Component
 public class HelmBakeManifestService extends BakeManifestService<HelmBakeManifestRequest> {
   private final HelmTemplateUtils helmTemplateUtils;
-  private static final String HELM_TYPE = "HELM2";
+  private static final String HELM2_TYPE = "HELM2";
```

Comment:
Instead of adding additional strings to keep track of this, it might be cleaner to do something like:
```
private static ImmutableSet<TemplateRenderer> supportedTemplates = ImutableSet.of(TemplateRenderer.HELM2, TemplateRenderer.HELM3)
```
then `handles` could be `supportedTemplates.contains(TemplateRenderer.fromString(type))`.

That just reduces the number of places we have strings `HELM*` floating around given that we already have an enum to represent it.

Reply:
Updated to use `ImmutableSet<String>` from the enum values, `TemplateRenderer.fromString(type)` is slightly more involved since it is not exception free.

Reply:
Looks good, thanks!  Ideally we'd maybe even make an exception-free version `fromString` so we could still use the enum, but this is definitely better as it keeps it more encapsulated.